### ISSUE TITLE
Light mode in gvim shows numbers in #C7F1FB until reload

### ISSUE THREAD

**Issue by zigford** [gruvbox-community/gruvbox#100]
On Windows I have only this in my Vimrc (For testing purposes)

```
set number
set background=light
set colorscheme gruvbox
```
Which looks like this:
![image](https://user-images.githubusercontent.com/16889643/58959685-ef644880-87e8-11e9-9403-fc5adb64f7e8.png)

After entering `:colorscheme gruvbox`, it looks like this:

![image](https://user-images.githubusercontent.com/16889643/58921946-408a2300-877c-11e9-915c-a8506bd3bcf7.png)

**Comment by zigford** [comment]
Dark mode doesn't exhibit the same issue, nor does this problem exist on Linux or MacOS

**Comment by rbong** [comment]
Thank you for the detailed issue. Unfortunately I tried to build Vim from scratch at the same version with the same .vimrc (although I had to use `colorscheme gruvbox` instead of `set colorscheme gruvbox`) and I could not recreate this.

Just one more detail that would help - what do you get when you enter in `:hi LineNr` and `:hi CursorLineNr`?

This workaround in your .vimrc might work for now: `autocmd VimEnter * colorscheme gruvbox`.

**Comment by zigford** [comment]
@rbong yes, `colorscheme gruvbox` is in my config, that was a mistake adding `set` in this issue.

```
:verbose hi LineNr
LineNr          xxx term=underline ctermfg=246 guifg=#a89984
Last set from ~\vimfiles\plugged\gruvbox\colors\gruvbox.vim
```
```
:verbose hi CursorLineNr
CursorLineNr    xxx term=bold ctermfg=136 ctermbg=223 guifg=#b57614 guibg=#ebdd2
Last set from ~\vimfiles\plugged\gruvbox\colors\gruvbox.vim
```
![image](https://user-images.githubusercontent.com/16889643/58935276-64fff280-87b0-11e9-8a2e-0b85c25d7f5e.png)

**Comment by rbong** [comment]
From those commands it seems as if the color is indeed set correctly. There is no background, but it should just show the `Normal` color behind the `LineNr`.

I tried some of the options from [this file](https://github.com/petrkle/vim-msi/blob/master/vim/_vimrc) and unfortunately still could not recreate. The only remotely related setting seems to be `colorscheme darkblue` but I can't see why that would do it. I also checked the vim git log `v8.1.1413..master` and did not see anything.

**Comment by zigford** [comment]
A bit more checking on my Vim installation. The version I had installed was not the MSI version, but the vim-tux package from chocolatey. It doesn't appear to ship with it's own _vimrc which is normally sourced before the user vimrc. 

I removed this version and installed the MSI version and the problem has gone away, so I am satisfied that it is not a gruvbox issue, but a quirk of the windows vim-tux package.
